orcestra_book/plans/HALO-20240909a.md

```diff
@@ -0,0 +1,176 @@
+---
+arrival_airport: TBPB
+categories: [ec_under, ec_track, c_north, c_mid, c_south, c_atr]
+crew:
+- job: PI
+  name: Julia Windmiller
+- job: WALES
+  name: tbd
+- job: HAMP
+  name: tbd
+- job: Dropsondes
+  name: tbd
+- job: Smart/VELOX
+  name: tbd
+- job: SpecMACS
+  name: tbd
+- job: Flight Documentation
+  name: tbd
+- job: Ground contact
+  name: tbd
+departure_airport: TBPB
+flight_id: HALO-20240909a
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+landing: '2024-09-09 19:50:00Z'
+orphan: true
+platform: HALO
+takeoff: '2024-09-09 10:40:00Z'
+---
+
+{logo}`PERCUSION`
+
+# Flight plan - {front}`flight_id`
+
+```{badges}
+```
+
+## Crew
+
+The flight is planned to take off at {front}`takeoff`.
+
+```{crew}
+```
+
+## Flight plan
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+from dataclasses import asdict
+from datetime import datetime
+import orcestra.sat
+from orcestra.flightplan import bco, sal, mindelo, point_on_track, LatLon, IntoCircle, FlightPlan
+
+# Some fixed coordinates
+
+lon_min, lon_max, lat_min, lat_max = -65, -5, -5, 25
+
+airport = bco
+
+radius = 72e3*1.852
+
+# Define dates for flight
+
+flight_time   = datetime(2024, 9, 9, 12, 0, 0)
+flight_index = f"HALO-{flight_time.strftime('%Y%m%d')}a"
+
+# Load satellite tracks 
+
+ec_fcst_time  = "2024-09-03"
+ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE",roi="BARBADOS") \
+    .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
+    .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
+
+# Create elements of track
+
+ec_north = point_on_track(ec_track,lat= 16.00).assign(label = "ec_north") 
+ec_south = point_on_track(ec_track,lat=  6.00).assign(label = "ec_south")
+c_north  = point_on_track(ec_track,lat= 12.50).assign(label = "c_north")
+c_south  = point_on_track(ec_track,lat=  8.00).assign(label = "c_south")
+c_mid    = point_on_track(ec_track,lat= 10.25).assign(label = "c_south")
+ec_under = point_on_track(ec_track, lat=14.0, with_time=True).assign(label = "ec_under", note = "meet EarthCARE")
+
+# Additional Waypoints
+
+extra_waypoints = []
+
```

Comment:
If there are no extra waypoints, you might want to skip them altogether. (I heard, that extra waypoint's aren't super encouraged anyways).

```suggestion
```